tests/queries/0_stateless/02731_formats_s3.sql

```diff
@@ -0,0 +1,13 @@
+-- Tags: no-fasttest
+-- Tag no-fasttest: Depends on AWS
+
+-- Reading from s3 a parquet file of size between ~1 MB and ~2 MB was broken at some point
+-- (bug in CachedOnDiskReadBufferFromFile).
+insert into function s3(s3_conn, filename='test_02731_parquet.parquet') select cityHash64(number) from numbers(170000) settings s3_truncate_on_insert=1;
```

Comment:
We can add this file in folder `data_minio` and don't generate it all the time (and test will be able to run in parallel)